src/ActualBlockEmitter.js

```diff
@@ -0,0 +1,26 @@
+const EventEmitter = require('events');
+const web3 = require('./utils/createAndUnlockWeb3');
+const logger = require('../src/config/winston');
+
+
+class ActualBlockEmitter extends EventEmitter {
```

Comment:
Why is it named `Actual`? Actual as opposed to what? I see that the event name is `actualBlock` but I can't see why either.

Reply:
It's about the block at front of blockchain.